_layouts/qrpage.html

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
```

Comment:
Should we keep the page layout here and using simpler layout (maybe without the image on the top and not the menu)?

![image](https://user-images.githubusercontent.com/1842467/234007465-1dbdd425-4ad8-4a77-9791-152dbdc38acb.png)

What do you think?

Reply:
Hmm. 🤔 I think the page may look a bit bland without the image...

Reply:
I think it is somehow the purpose, like linktree: 
![image](https://user-images.githubusercontent.com/1842467/234010544-c382376f-8c08-459b-8419-9d4f25b64325.png)

Reply:
Fair point. 😃 So, we remove the extra details from the frontmatter, yes?

Reply:
Using the `base` layout makes the page look very strange. Some of the content is hidden...

Reply:
I think it looks pretty neat now. What say you?